### ISSUE TITLE
RFE: Option to specify working directory other than /tmp to FOG installer (installfog.sh)

### ISSUE THREAD

**Issue by ggiesen** [FOGProject/fogproject#458]
As a security measure on all of our machines, we mount /tmp with `noexec`. This causes the FOG installer to fail while trying to run ./configure for UDPCast:

```
udpcast-20120424/config.h.in
udpcast-20120424/udpcast.c
udpcast-20120424/usage.h
udpcast-20120424/auto-rate.h
../lib/common/functions.sh: line 443: ./configure: Permission denied
...skipping...
```

At least one of the offending lines is here.

https://github.com/FOGProject/fogproject/blob/171d63724131c396029992730660497d48410842/lib/common/functions.sh#L443

I tried prepending the line with `sh` but the installer later fails on:

```
checking whether we are cross compiling... configure: error: in `/tmp/udpcast-20120424':
configure: error: cannot run C compiled programs.
If you meant to cross compile, use `--host'.
```

It would be nice if we could specify a working directory for the FOG installer other than /tmp.

**Comment by Sebastian-Roth** [comment]
@ggiesen Hi Gary, thanks for your input on this. I see why this is causing trouble for you.

We actually use `/path/to/fogproject-code/tmp` for certain things like temporary SQL scripts, downloading inits and kernels. I think we could easily use that directory for udpcast compilation as well. What do you think?

Sure making it configurable makes it more flexible but it just adds more complexity where I don't think it's really needed.

**Comment by ggiesen** [comment]
> @ggiesen Hi Gary, thanks for your input on this. I see why this is causing trouble for you.
> 
> We actually use `/path/to/fogproject-code/tmp` for certain things like temporary SQL scripts, downloading inits and kernels. I think we could easily use that directory for udpcast compilation as well. What do you think?
> 
> Sure making it configurable makes it more flexible but it just adds more complexity where I don't think it's really needed.

Yes as I started poking around into the scripts, I saw that was the case. I think that makes the most sense, as whereever you untar the tarball is where it will execute from.

**Comment by Sebastian-Roth** [comment]
@ggiesen This is fixed in the latest `dev-branch` version. See commit above.
